Developer tools/Useful software.md

```diff
@@ -0,0 +1,23 @@
+In the following list, we'll give some useful software and links that will speed up your development.
+
+[SVG 2 WP](https://svg-2-wp.goranalkovic.com/) - a super useful site created by Goran Alković, that will transform your SVG in a format compatible with JS use (either in manifest or JS files)

```

Comment:
Some sentences end with a **dot** others don't, maybe to keep it consistent one way or another